spec/periodicChecksSpec.js

```diff
@@ -856,5 +856,89 @@ describe('Periodic Checks Module', () => {
         });
       });
     });
+
+    describe('Should Check If a Opened Issue is Assigned to a project', () => {
+      beforeEach(async () => {
+        github.issues.listForRepo = jasmine
+          .createSpy('listForRepo')
+          .and.resolveTo({
+            data: [
+              issues.withProject,
+              issues.anotherWithProject,
+              issues.withoutProject,
+            ],
+          });
+        payloadData.name = 'issues';
+        payloadData.payload.action = 'opened';
+        spyOn(periodicCheckModule, 'ensureNewIssuesHaveProjects').
+          and.callThrough();
+        await robot.receive(payloadData);
```

Comment:
The `payloadData` should contain some information about the issue. See the `issue.assigned.json` payload.
If possible, use that as the payload and set the action to be opened.

Reply:
Do this here and in the other test.

Reply:
Working on it. Thanks.